build-logic/module-identity/src/main/kotlin/gradlebuild/identity/extension/ModuleIdentityExtension.kt

```diff
@@ -35,7 +35,19 @@ abstract class ModuleIdentityExtension(val tasks: TaskContainer, val objects: Ob
     abstract val snapshot: Property<Boolean>
     abstract val promotionBuild: Property<Boolean>
 
+    /**
+     * The actual build branch.
+     */
     abstract val gradleBuildBranch: Property<String>
+
+    /**
+     * The base branch for pre-tested commit, i.e. the branch is not `master`/`release`,
+     * but it's tested in the same way of `master`/`release`.
+     *
+     * The pre-tested commit branch name looks like "pre-test/master/queue/alice/personal-branch".
+     */
+    abstract val preTestedCommitBaseBranch: Property<String>
```

Comment:
Why is this the called `preTestedCommitBaseBranch`. How about calling it `logicalBranch` or something like that? A pre-tested commit branch is/will be the same as `master`, right? Or `master` would be the pre-tested commit branch without doing pre-tested commit.